docs/build.gradle

```diff
@@ -15,8 +15,13 @@
  */
 
 plugins {
-  id 'org.asciidoctor.convert' version '1.5.7'
+  id 'org.asciidoctor.convert' version '1.5.3'
```

Comment:
Hehe... there is a bug in asciidoctorj when used with jruby that prevents it from loading the asciidoctor-diagram gem properly.  This is the only way I managed to get everything working:

https://github.com/asciidoctor/asciidoctor-diagram/issues/159 (closed because it's not an asciidoctor-diagram bug)
https://github.com/asciidoctor/asciidoctorj/issues/562

Reply:
Sigh...